tests/xilinx/latches.ys

```diff
@@ -0,0 +1,16 @@
+read_verilog latches.v
+
+proc
+flatten
+equiv_opt -assert -run :prove -map +/xilinx/cells_sim.v synth_xilinx # equivalency check
+async2sync
+equiv_opt -assert -run prove: -map +/xilinx/cells_sim.v synth_xilinx # equivalency check
+design -load postopt # load the post-opt design (otherwise equiv_opt loads the pre-opt design)
+
+design -load preopt
+synth_xilinx
+cd top
+select -assert-count 1 t:LUT1
```

Comment:
This part is worrying — instead of getting an inverter on the G pin, we should just get LDCE_1 used here.  I'll investigate it...